javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,18 @@
+var fullName = 'Michaela Mauriello';
+console.log('value of fullName: ' + fullName);
+var isCool = true;
+console.log('value of isCool: ' + isCool);
+var totalPets = 2;
+console.log('value of totalPets: ' + totalPets);
+
+console.log('typeof fullName: ' + typeof 'Michaela Mauriello');
+console.log('typeof isCool: ' + typeof true);
+console.log('typeof totalPets: ' + typeof 2);
```

Comment:
Instead of typing the value in your code directly, you should use the variable that holds that value.